www/hockeyapp.js

```diff
@@ -20,6 +20,9 @@ var hockeyapp = {
     feedback: function (success, failure) {
         exec(success, failure, "HockeyApp", "feedback", []);
     },
+    feedbackModal: function (success, failure, takeScreenshot, data) {
```

Comment:
What do you think of the name `composeFeedback` or something similar? The use of `Modal` here seems potentially confusing, since the `feedback` method displays the feedback listing in a modal fashion as well, so it's not really the modality that differentiates the two methods, but rather, the view they display: list vs. compose screen.